internal/test/wmcb/wmcb_test.go

```diff
@@ -112,10 +90,14 @@ func TestWMCB(t *testing.T) {
 	for _, vm := range framework.WinVMs {
 		log.Printf("Testing VM: %s", vm.GetCredentials().InstanceId())
 		wVM := &wmcbVM{vm}
-		files := strings.Split(*filesToBeTransferred, ",")
-		for _, file := range files {
-			err := wVM.CopyFile(file, remoteDir)
-			require.NoError(t, err, "error copying %s to the Windows VM", file)
+		folders := strings.Split(*foldersToBeTransferred, ",")
```

Comment:
`folder` and `directory` are used interchangeably here. Should we just go with `directory` / `dir`?